R/get_timescale_data.R

```diff
@@ -0,0 +1,107 @@
+#' Get geological timescale data
+#'
+#' This function takes a name of a geological timescale and returns data for the
+#' timescale. Valid names include those of built-in `data.frames` ([GTS2020()]
+#' and [GTS2012()]) and exact matches to those hosted by Macrostrat (see full
+#' list here: <https://macrostrat.org/api/defs/timescales?all>). If a built-in
```

Comment:
Maybe in the details section we can mention that GTS in Macrostrat is always the most recent? I am also wondering if users should also just be able to enter "Stages", "Periods", etc in name?

Reply:
Hmm...maybe...I'd be fine listing a few Macrostrat timescales that we think are useful, but I don't think we should get too detailed because things can change without much notice.